src/main.ts

```diff
@@ -7,100 +7,104 @@ export interface Filter {
   value: string | number | boolean;
 }
 
-export interface PluginMeta
-  extends Meta<{
+export type PluginMeta = Meta<{
     config: {
       eventsToDrop?: string;
       keepUndefinedProperties?: "Yes" | "No";
     };
     global: {
-      filters: Filter[];
+      filters: Filter[][];
```

Comment:
This would be a breaking change which would break all existing integrations...

The only way we could merge something like this is to have it support both types or modify the type system so that it could support both the list and the nested list...

I also am not sure that the nested list is particularly clear... In our API we often have a "filterGroup" type that has concepts to define how the groups work. That might be a bit much for this plugin though...

Reply:
Please let me know If i can help with any way as this very time critical for us <3

I have updated the following plugin. I created a function that reads either an array or an array of arrays and matches it to the new format.

This function is used in two places:
- **Setup Plugin** (in case someone has a JSON file locally from a while ago)
- **Process Events** (for people who are using an older version).

I reverted my changes for the old tests to use an array, while new tests have been added for the OR logic (array of arrays).